### PR TITLE
add test for re-rendering query builder result panel

### PR DESCRIPTION
**Pull request by YannanGao-gs**
## Summary

add test for query builder result panel

## How did you test this change?

- [x] Test(s) added
- [ ] Manual testing (please provide screenshots/recordings)
- [ ] No testing (please provide an explanation)

### CODE REVIEW

File: packages/legend-manual-tests/src/__tests__/query-execution/TEMPORARY__QueryBuilderExecution.test.tsx [hunk 1]
```diff
@@ -183,4 +183,78 @@ test(integrationTest('test query execution with parameters'), async () => {
...
+  await act(async () => {
+    fireEvent.click(getByText(queryBuilderResultPanel1, 'Run Query'));
+  });
+  const parameterValueDialog1 = await waitFor(() =>
+    renderResult.getByRole('dialog'),
+  );
+  await waitFor(() =>
+    fireEvent.change(
+      guaranteeNonNullable(
+        parameterValueDialog1.getElementsByClassName(
```
Comment (MauricioUyaguari) [inline packages/legend-manual-tests/src/__tests__/query-execution/TEMPORARY__QueryBuilderExecution.test.tsx:239]: shouldnt there be only one input in the dialog? dont really like querying by classname as we can refactor that easily.

File: packages/legend-manual-tests/src/__tests__/query-execution/TEMPORARY__QueryBuilderExecution.test.tsx [hunk 2]
```diff
@@ -183,4 +183,78 @@ test(integrationTest('test query execution with parameters'), async () => {
...
+      ),
+      {
+        target: { value: 20 },
+      },
+    ),
+  );
+  await waitFor(() => fireEvent.click(getByText(parameterValueDialog1, 'Run')));
+
+  expect(
+    queryBuilderResultPanel1.getElementsByClassName(
```
Comment (MauricioUyaguari) [inline packages/legend-manual-tests/src/__tests__/query-execution/TEMPORARY__QueryBuilderExecution.test.tsx:251]: same thing here, could we query maybe by testId? add a testID on the result panel?

File: packages/legend-manual-tests/src/__tests__/query-execution/TEMPORARY__QueryBuilderExecution.test.tsx (hunk 1 above)
Comment (YannanGao-gs) [reply packages/legend-manual-tests/src/__tests__/query-execution/TEMPORARY__QueryBuilderExecution.test.tsx]: I deleted this as the previous parameter value is saved so it doesn't need to be reset.

Comment (YannanGao-gs) [reply packages/legend-manual-tests/src/__tests__/query-execution/TEMPORARY__QueryBuilderExecution.test.tsx]: however, I used this `getElementsByClassName ` the first time I set the value

**Review by MauricioUyaguari — APPROVED** [review]

**Pull request merged**